demo/components/AppNavigationBar.vue

```diff
@@ -1,9 +1,9 @@
 <template>
-  <p-navigation-bar class="app-navigation-bar" :class="classes.root" v-bind="{ layout }">
+  <p-navigation-bar class="app-navigation-bar p-background" :class="classes.root" v-bind="{ layout }">
```

Comment:
Try to keep only BEM classes in template elements, even demo ones; that gives us very clear places to maintain styles